lib/govuk_document_types.rb

```diff
@@ -18,4 +19,9 @@ def self.supertypes(document_type:)
 
     types
   end
+
+  def self.supergroups(ids:)
```

Comment:
Just wondering if this needs to be a key, as there's only one argument?

Reply:
Yeah that's a fair point, I've gone with a method signature similar to `.supertypes` purely for convention and conformity.